OsmAnd-shared/src/commonMain/kotlin/net/osmand/shared/gpx/GpxTrackAnalysis.kt

```diff
@@ -24,7 +24,6 @@ class GpxTrackAnalysis {
 	}
 
 	var name: String? = null
-	var totalDistanceWithoutGaps = 0f
 	var timeSpanWithoutGaps: Long = 0
 	var expectedRouteDuration: Long = 0
 	var timeMovingWithoutGaps: Long = 0
```

Comment:
Do we have the same issue with other variables here?

Reply:
have not faced such problems with others.